_posts/2023-07-29-oauth-authentication-scala-http4s.md

```diff
@@ -0,0 +1,672 @@
+---
+title: "Oauth Authentication in Scala with Http4s"
+date: 2023-07-27
+header:
+  image: "/images/blog cover.jpg"
+tags: []
+excerpt: "Learn about OAuth authentication using Http4s."
+toc: true
+toc_label: "In this article"
+---
+
+## Introduction
```

Comment:
Let's do the same chapter number scheme as last time
h2s are "1.", "2." ...
h3s are "1.1.", "1.2." etc

Reply:
Updated